src/index.html

```diff
@@ -4,9 +4,388 @@
+            >
+                Ноутбуки и компьютеры
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#gadgets">Гаджеты</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#tablets">Планшеты</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#photo">ФОТО</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#video">ВИДЕО</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog">
+      <div class="product-card catalog__product-card" data-qa="card">
+        <img
+          class="product-card__picture"
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac"
+        >
+
+        <h3 class="product-card__title">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </h3>
+
+        <span class="product-card__code">
+          Код товара: 195434
+        </span>
+
+        <div class="product-card__reviews">
+          <div class="stars stars--4">
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+          </div>
+
+          <span class="product-card__review-text">Отзывов: 5</span>
+        </div>
+
+        <div class="product-card__price">
+          <span class="product-card__price-text">Цена:</span>
+          <span class="product-card__price-value">69 999 грн</span>
+        </div>
+
+        <a
+          href="#195434"
+          class="product-card__buy-button"
+          data-qa="card-hover"
+        >
+          Купить
+        </a>
+      </div>
+
+
+      <div class="product-card catalog__product-card">
+        <img
+          class="product-card__picture"
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac"
+        >
+
+        <h3 class="product-card__title">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </h3>
+
+        <span class="product-card__code">
+          Код товара: 195434
+        </span>
+
+        <div class="product-card__reviews">
+          <div class="stars stars--4">
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+          </div>
+
+          <span class="product-card__review-text">Отзывов: 5</span>
+        </div>
+
+        <div class="product-card__price">
+          <span class="product-card__price-text">Цена:</span>
+          <span class="product-card__price-value">69 999 грн</span>
+        </div>
+
+        <a
+          href="#195434"
+          class="product-card__buy-button">
+          Купить
+        </a>
```

Comment:
Fix it in all cards.
```suggestion
        <a
          href="#195434"
          class="product-card__buy-button"
        >
          Купить
        </a>
```